Makefile

```diff
@@ -55,12 +55,4 @@ submit-today:
 bundler:
 	bundle check>/dev/null || bundle install
 
-docker-build-image:
```

Comment:
Don't we want to keep this? So if someone wants to build a Docker image, it's still easy.

If we don't want to keep Docker, shouldn't we delete the `Dockerfile` too? (or at least the `TAG ?=` definition in this Makefile)